modules/dkan/dkan_dataset/dkan_dataset.module

```diff
@@ -1304,7 +1304,21 @@ function dkan_dataset_modified_date($node) {
 
   if (isset($node->field_harvest_source_modified) &&
     !empty($node->field_harvest_source_modified->value())) {
-    $date = format_date($node->field_harvest_source_modified->value(), 'iso_8601_date');
+    // Check if the value is a valid date or term.
+    $d = validateDate($node->field_harvest_source_modified->value());
+    $terms = array_keys(dkan_dataset_content_types_iso_frecuency_map());
```

Comment:
This isn't sufficient to validate ISO 8601 strings as specified for the modified field in the POD schema. 

> Dates should be ISO 8601 of highest resolution. In other words, as much of `YYYY-MM-DDThh:mm:ss.sTZD` as is relevant to this dataset. If there is a need to reflect that the dataset is continually updated, ISO 8601 formatting can account for this with repeating intervals. For instance, `R/P1D` for daily, `R/P2W` for every two weeks, and `R/PT5M` for every five minutes.

The limited values we provide for Frequency wouldn't support all of the possibilities there.

We should be able to parse it by checking if it starts with `R/` and then grabbing the string after that and running it through PHP's `DateInterval::format()`. See http://us1.php.net/manual/en/class.dateinterval.php

Reply:
@dafeder also this sounds like we would just display the 'R/PT5M' value on the dataset and not 'Every 5 minutes' then?